OpenRA.Game/Selection.cs

```diff
@@ -14,19 +14,29 @@
 using OpenRA.Graphics;
 using OpenRA.Primitives;
 using OpenRA.Traits;
+using System;
 
 namespace OpenRA
 {
 	public class Selection
 	{
+		readonly List<Action<Selection>> onSelectionChanged = new List<Action<Selection>>();
```

Comment:
This should be a `public event Action` for consistency with our other event handlers.